.gitlab/docker/Dockerfile.spack_build

```
@@ -15,7 +15,8 @@ COPY . ${SOURCE_DIR}
 RUN spack spec -lI $spack_spec
 
 # Configure & Build
-RUN spack build-env $spack_spec -- bash -c "cmake -B${BUILD_DIR} ${SOURCE_DIR} \
+RUN spack env activate --sh . && \
+    spack build-env $spack_spec -- bash -c "cmake -B${BUILD_DIR} ${SOURCE_DIR} \
```

```suggestion
RUN spack -e . build-env $spack_spec -- bash -c "cmake -B${BUILD_DIR} ${SOURCE_DIR} \
```
It might also make sense to give the environment a more memorable directory than the current directory (is that `/` currently?).